texastribune/donations
# Giving Tuesday 2023

#### What's this PR do?
- Add Giving Tuesday copy
- Updates the thermometer to include dollars instead of member count and Giving Tuesday branding

#### Why are we doing this? How does it help us?

Helps bring in end of year donations

#### How should this be manually tested?

Kassie has approved this but basically open on the donate page and confirm that the thermometer displays and the yellow box messaging looks good

#### How should this change be communicated to end users?

Let Kassie know when this is merged

#### Are there any smells or added technical debt to note?

Nah

#### What are the relevant tickets?

https://airtable.com/appyo1zuQd8f4hBVx/tbloNZu8GkM52NKFR/viwb2L2CHc4ZWqkC5/recv0ecHBKAMWD1SX?blocks=hide

#### Have you done the following, if applicable:
***(optional: add explanation between parentheses)***


#### TODOs / next steps:

* [x] *turn thermometer on*
* [ ] *merge this Tuesday morning*
* [ ] *bust social cache on facebook and twitter*
